examples/eventloop.py

```diff
@@ -13,10 +12,7 @@
 
 
 async def main() -> None:
-    notifier = DesktopNotifier(
-        app_name="Sample App",
-        notification_limit=10,
```

Comment:
not a feature i actually use but curious what is the motivation for removing this? just decreased complexity or something else?